docs/user-manual/gaussian-splatting/building/engine-features/unified-rendering.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Unified Rendering
+---
+
+Unified Rendering is a beta feature that significantly enhances the capabilities of the `GSplatComponent` by enabling global sorting and Level of Detail (LOD) streaming for Gaussian Splats.
+
+:::info Beta Feature
+
+Unified Rendering is currently in beta. If you encounter any issues, please report them on the [PlayCanvas Engine GitHub repository](https://github.com/playcanvas/engine/issues).
+
+:::
+
+## Enabling Unified Rendering
+
+To enable unified rendering on a GSplat component, set the [`unified`](https://api.playcanvas.com/engine/classes/GSplatComponent.html#unified) property to `true`:
+
+```javascript
+entity.gsplat.unified = true;
+```
+
+:::note
+
+The `unified` property can only be changed when the component is disabled.
```

Comment:
I didn't know about this restriction. I think it's a similar deal with light and collision types but they just handle it internally. Can we do that here?